proj/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,38 @@
+# The GeoRust Code of Conduct
```

Comment:
It might make sense for this to be deduplicated at the repo root, rather than in each crate.

Also, I think we have to merge our ci config in to a single `.travis.yml` at the repo root, right? 

And another option would be to leave the proj code where it was, at the repo root, and add the `workspace` declaring `proj-sys` to proj's Cargo.toml, like the gdal crate: https://github.com/georust/gdal/blob/master/Cargo.toml#L33 

not a big deal either way - might make git history a little easier to track.

If you want to do any of these, but are too busy, you can feel free to punt it to me. I have a sadistic affinity for all things CI.

Reply:
Ha! I think I prefer the hierarchical model with the original proj location – will shuffle things around, tidy, and unify the CI tomorrow.